modular_splurt/code/modules/mob/living/carbon/human/species_types/teshari.dm

```diff
@@ -0,0 +1,192 @@
+	exotic_blood_color = "#D514F7"
+	disliked_food = GROSS | GRAIN
+	liked_food = MEAT
+	payday_modifier = 0.75
+	attack_verb = "claw"
+	attack_sound = 'sound/weapons/slash.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	ass_image = 'icons/ass/asslizard.png'
+
+	species_traits = list(MUTCOLORS,
+		EYECOLOR,
+		NO_UNDERWEAR,
+		HAIR,
+		FACEHAIR
+		)
+
+	coldmod = TESHARI_COLDMOD
+	heatmod = TESHARI_HEATMOD
+	brutemod = TESHARI_BRUTEMOD
+	burnmod = TESHARI_BURNMOD
+
+
+/datum/species/mammal/teshari/on_species_gain(mob/living/carbon/human/C, datum/species/old_species, pref_load)
+	if(ishuman(C))
+		C.verbs += /mob/living/carbon/human/proc/sonar_ping
+		C.verbs += /mob/living/carbon/human/proc/hide
+		C.setMaxHealth(50)
+		C.physiology.hunger_mod *= 2
+		C.add_movespeed_modifier(/datum/movespeed_modifier/teshari)
+
+/datum/movespeed_modifier/teshari
+	multiplicative_slowdown = -0.2
+
+/datum/emote/sound/teshari
+	mob_type_allowed_typecache = list(/mob/living/carbon/human)
+	emote_type = EMOTE_AUDIBLE
+
+/datum/language_holder/teshari
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/schechi = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/schechi = list(LANGUAGE_ATOM))
+
+/datum/language/schechi
+
+/datum/emote/living/audio/teshari/teshsqueak
+	key = "surprised"
+	key_third_person = "surprised"
+	message = "chirps in surprise!"
+	message_mime = "lets out an <b>inaudible</b> chirp!"
+	emote_sound = 'modular_splurt/sound/voice/teshsqueak.ogg' // Copyright CC BY 3.0 InspectorJ (freesound.org) for the source audio.
+	emote_cooldown = 2.1 SECONDS
+
+/datum/emote/living/audio/teshari/teshsqueak/run_emote(mob/user, params)
+	var/datum/dna/D = user.has_dna()
+	if(D.species.name != "Teshari")
+		return
+	// Return normally
+	. = ..()
+
+/datum/emote/living/audio/teshari/teshchirp
+	key = "tchirp"
+	key_third_person = "tchirp"
+	message = "chirps!"
+	message_mime = "lets out an <b>inaudible</b> chirp!"
+	emote_sound = 'modular_splurt/sound/voice/teshchirp.ogg' // Copyright CC BY 3.0 InspectorJ (freesound.org) for the source audio.
+	emote_cooldown = 2.1 SECONDS
+
+/datum/emote/living/audio/teshari/teshchirp/run_emote(mob/user, params)
+	var/datum/dna/D = user.has_dna()
+	if(D.species.name != "Teshari")
+		return
+	// Return normally
+	. = ..()
+
+/datum/emote/living/audio/teshari/trill
+	key = "trill"
+	key_third_person = "trill"
+	message = "trills!"
+	message_mime = "lets out an <b>inaudible</b> chirp!"
+	emote_sound = 'modular_splurt/sound/voice/teshtrill.ogg' // Copyright CC BY 3.0 InspectorJ (freesound.org) for the source audio.
+	emote_cooldown = 2.1 SECONDS
+
+/datum/emote/living/audio/teshari/trill/run_emote(mob/user, params)
+	var/datum/dna/D = user.has_dna()
+	if(D.species.name != "Teshari")
+		return
+	// Return normally
+	. = ..()
+
+/datum/emote/living/audio/teshari/teshscream
+	key = "teshscream"
```

Comment:
Can you assign previous emotes the same way here? Not a huge deal otherwise.
(Also, I haven't checked but I thought you could assign scream sound per species without adding a separate emote?)